vllm/v1/executor/multiproc_executor.py

```diff
@@ -617,5 +640,4 @@ def worker_busy_loop(self):
                 continue
 
             if output_rank is None or self.rank == output_rank:
-                self.worker_response_mq.enqueue(
-                    (WorkerProc.ResponseStatus.SUCCESS, output))
+                self.async_output_queue.put(output)
```

Comment:
I think we should ensure that the failure responses also propagate via the same queue/thread (see exception block above), to ensure correct ordering.